src/main/resources/reference.conf

```diff
@@ -76,6 +79,18 @@ cassandra-journal {
   # The time to wait before cassandra will remove the thombstones created for deleted entries.
   # cfr. gc_grace_seconds table property documentation on http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
   gc-grace-seconds = 864000
+  
+  # Configure know tags in this section
```

Comment:
What is reason for this?

This seems to be imposing artificial restrictions on user that don't have substantiation other than technical limitations. I think we should be able to allow any number of tags in any order. That of course complicates replay etc., but should be technically feasible. At least that was my idea of the design. Or am I missing something?